include/service/entities/password_reset_service.js

```diff
@@ -96,15 +96,15 @@ module.exports = function(pb) {
             }
 
             //attempt to retrieve any existing reset
-            self.getSingle({where: {userId: userObj.id}}, function(err, passwordResetObj) {
+            self.getSingle({where: {user_id: userObj[pb.DAO.getIdField()].toString()}}, function(err, passwordResetObj) {
```

Comment:
When we recently changed this service the plan to was eliminate the underscore.  Was there one that we missed?